Patches/Dragonian Race/ThingDefs_Misc/DR_Weapon.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+  <Operation Class="PatchOperationSequence">
+	<operations>
+	  <li Class="PatchOperationFindMod">
+			
+		<mods><li>Dragonian Race</li></mods>
+			
+		<match Class="PatchOperationSequence">
+		<operations>
+			
+			<!--Base ver is pretty much comparable to a zeushammer (slightly less damage on swing, more damage on poke, better EMP effect)-->
+			
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName="DR_BattleHammer"]/tools</xpath>
+				<value>
+					<tools>
+						<li Class="CombatExtended.ToolCE">
+							<label>handle</label>
+							<capacities>
+								<li>Poke</li>
+							</capacities>
+							<power>5</power>
+							<extraMeleeDamages>
```

Comment:
The original weapon in this instance addtionally zaps people with the handle poke as well, rather than just with the head like with the Zeushammer.

Given the way secondary damage works, it is admittedly largely redundant since the majority of the time it won't matter (since secondary damage doesn't seem to go off if the main damage gets blocked).